packages/core-data/src/entities.js

```diff
@@ -154,6 +154,14 @@ export const defaultEntities = [
 		baseURLParams: { context: 'edit' },
 		key: 'stylesheet',
 	},
+	{
+		label: __( 'Plugins' ),
+		name: 'plugin',
+		kind: 'root',
+		baseURL: '/wp/v2/plugins',
+		baseURLParams: { context: 'edit' },
+		key: 'plugin',
+	},
```

Comment:
I tested this out in the console. `wp.data.select( 'core' ).getPlugin( 'gutenberg/gutenberg' )` works ok. Though I don't know why it has to be 'gutenberg/gutenberg' instead of just plain 'gutenberg'. A concern is that this endpoint isn't going to be useful if a plugin doesn't identify itself correctly in the template data.

`wp.data.select( 'core' ).getPlugins()` doesn't work. It just shows the first plugin multiple times. Not sure why.

Reply:
> Though I don't know why it has to be 'gutenberg/gutenberg' instead of just plain 'gutenberg'.

Just a quirk of how WordPress identifies plugins by which PHP file in `wp-content/plugins` contains the plugin header comment. Gutenberg's is `gutenberg/gutenberg.php`. There's nothing preventing you from having two completely seperate plugins at `gutenberg/gutenberg.php` and `gutenberg.php`. The REST API removes the `.php` at the end because it's irrelevant to API consumers.

> wp.data.select( 'core' ).getPlugins() doesn't work. It just shows the first plugin multiple times. Not sure why.

I ran this locally and it seems okay to me.

<img width="622" alt="Screen Shot 2021-11-25 at 3 49 27 pm" src="https://user-images.githubusercontent.com/612155/143381375-64ace79c-1217-4efa-83d3-3aa08f48e8bb.png">

Reply:
Works for me now too. Not sure what it was.

Reply:
I have concerns this isn't going to work because of the need for 'gutenberg/gutenberg' as the identifier.

Maybe it's better to look at querying the plugin directory. For now it falls back to displaying the slug, which I think is ok.